db/migrate/20140804102641_add_email_to_broadcast_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToBroadcastMessages < ActiveRecord::Migration
+  def change
+    add_column :broadcast_messages, :email, :boolean
```

Comment:
You've added this DB column, but I don't see it used anywhere else. Is this really needed?